### PR TITLE
Add table settings support

### PR DESCRIPTION
**Pull request by ilchuk96**

### CODE REVIEW

**Review by LuckySting — CHANGES_REQUESTED** [review]
Almost done 🦕

File: ydb_sqlalchemy/sqlalchemy/__init__.py [hunk 1]
```diff
@@ -343,7 +343,35 @@ def get_bind_types(
 
 
 class YqlDDLCompiler(DDLCompiler):
-    pass
+    def post_create_table(self, table):
```
[the remark this answers is not in the document]
Comment (ilchuk96) [reply ydb_sqlalchemy/sqlalchemy/__init__.py]: fixed

File: ydb_sqlalchemy/sqlalchemy/__init__.py [hunk 2]
```diff
@@ -343,7 +343,35 @@ def get_bind_types(
...
+        ydb_opts = table.dialect_options["ydb"]
+        with_content = []
```
[the remark this answers is not in the document]
Comment (ilchuk96) [reply ydb_sqlalchemy/sqlalchemy/__init__.py]: renamed

File: ydb_sqlalchemy/sqlalchemy/__init__.py (hunk 1 above)
[the remark this answers is not in the document]
Comment (ilchuk96) [reply ydb_sqlalchemy/sqlalchemy/__init__.py]: fixed

**Comment by LuckySting** [comment]
Looks good for me, let's remove "WIP" prefix

**Review by LuckySting — APPROVED** [review]

**Review by LuckySting — APPROVED** [review]

File: test/test_core.py [hunk 3]
```diff
@@ -200,3 +203,164 @@ def test_select_types(self, connection):
...
+    def _create_table_and_get_desc(connection, metadata, **kwargs):
+        table = Table(
+            "clause_with_test",
+            metadata,
+            Column("id", types.UInt32, primary_key=True),
+            **kwargs,
+        )
+        table.create(connection)
+
+        session: ydb.Session = connection.connection.driver_connection.pool.acquire()
```
Comment (LuckySting) [inline test/test_core.py:221]: **nitpick**: We have to return session to the pool here, but it's not crucial in tests I think

Comment (ilchuk96) [reply test/test_core.py:222]: Added calling of `delete()` method for session

File: test/test_core.py [hunk 4]
```diff
@@ -200,3 +203,164 @@ def test_select_types(self, connection):
...
+        return session.describe_table("/local/" + table.name)
+
+    @pytest.mark.parametrize(
+        "auto_partitioning_by_size,res",
+        [
+            (None, 1),
```
[the remark this answers is not in the document]
Comment (ilchuk96) [reply test/test_core.py]: Did I add it correctly?
```
from ydb._grpc.v4.protos import ydb_common_pb2
<...>
ydb_common_pb2.FeatureFlag.Status.ENABLED
```

**Review by rekby — APPROVED** [review]
Thanks for the work

**Pull request merged**